packages/card/src/vaadin-card.js

```diff
@@ -279,10 +280,46 @@ class Card extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
     `;
   }
 
+  static get properties() {
+    return {
+      /**
+       * The title of the card. When set, any custom slotted title is removed and this string-based title is used instead. If this title is used, an `aria-labelledby` attribute that points to the generated title element is set.
+       *
+       * @type {String}
+       */
+      title: {
+        type: String,
+        notify: true,
+        observer: '__titleChanged',
+      },
+
+      /**
+       * Sets the heading level (`aria-level`) for the string-based title. Valid values are from 1 through 6. If not set or an invalid value is used, the level defaults to 2.
+       *
+       * @type {Number}
+       */
+      __titleHeadingLevel: {
```

Comment:
With this property name, setting heading level with HTML attribute doesn't work:

```html
<vaadin-card class="playground" title="Card" title-heading-level="3"></vaadin-card>
```

Reply:
I assumed we weren't going to use it as an attribute, but only as a property. Do you think we should?

Reply:
Set `reflectToAttribute` `true` for the property.